src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "name": "Google BigQuery API",
+    "documentation": "https://cloud.google.com/bigquery/docs/reference/rest/v2/",
```

Comment:
sounds like there's three interesting levels of documentation

product documentation - what it does
java client documentation - getting started, samples, etc
java client API reference documentation - javadoc